frontend/src/components/Search/SearchForm/SearchForm.tsx

```diff
@@ -509,12 +550,6 @@ export const Search: React.FC<{
 		startDate,
 	])
 
-	useEffect(() => {
-		// necessary to update the combobox with the URL state
```

Comment:
Don't believe this is necessary with the new search context

Reply:
I think you're right. Seems to work fine on page reload 👍

Reply:
Added back since was causing the bug above